.github/workflows/main.yml

```
@@ -0,0 +1,41 @@
+name: Tests
+'on':
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  Build:
+    runs-on: '${{ matrix.os }}'
+    strategy:
+      matrix:
+        # ToDo enable more OS's []
+        os: ["ubuntu-latest"]
+        python-version: ['3.8', '3.9']
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
```

Use the version from the matrix!
```suggestion
        python-version: ${{ matrix.python-version }}
```

---

https://github.com/improbable-eng/perforce-buildkite-plugin/runs/4780538941?check_suite_focus=true#step:3:6